Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -574,16 +586,29 @@ open class MapView: UIView {
         displayLink?.invalidate()
         displayLink = nil
 
-        guard let window = window else { return }
+        guard let window = window else {
+            cameraAnimatorsRunner.cancelAnimations()
+            cameraAnimatorsRunnerEnablable.isEnabled = false
+            return
+        }
 
         displayLink = dependencyProvider.makeDisplayLink(
             window: window,
             target: ForwardingDisplayLinkTarget { [weak self] in
                 self?.updateFromDisplayLink(displayLink: $0)
             },
             selector: #selector(ForwardingDisplayLinkTarget.update(with:)))
+
+        guard let displayLink = displayLink else {
+            cameraAnimatorsRunner.cancelAnimations()
+            cameraAnimatorsRunnerEnablable.isEnabled = false
+            return
```

Comment:
https://github.com/mapbox/mapbox-maps-ios/pull/1200/files#diff-5be8ad5aa3482beb6c6ec989ea84a88e38d60e6a294963b0051c9b9f3eddedbeR613
this line will not be executed now if `displayLink`is nil, if I am not mistaken the previous implementation will still execute this line and subscribe MapView to life cycle events. Is this change intended?